apps/glusterfs/placer_arbiter.go

```diff
@@ -54,10 +51,10 @@ func newArbiterOpts(opts PlacementOpts) *arbiterOpts {
 	}
 }
 
-func (aopts *arbiterOpts) discount(index int) (err error) {
+func (aopts *arbiterOpts) discount(index int, averageFileSize uint64) (err error) {
```

Comment:
You shouldn't need to make it an extra argument here. arbiterOpts refers back to the placement opts which you already enhanced with the AverageFileSize method. So, something like:
```golang
func (aopts *arbiterOpts) discount(index int) (err error) {
 	if index == arbiter_index {
 		aopts.brickSize, err = discountBrickSize(
			aopts.brickSize, aopts.o.AverageFileSize())
 	}
 	return
}
```
should work.

Reply:
That is nice, I missed it. Updated.